security/web-security.md

```diff
@@ -34,30 +35,39 @@ Apprentices will learn secure development basics, common pitfalls, and how to av
 - [Validator](https://github.com/chriso/validator.js)
 - [Parsley, the ultimate JavaScript form validation library](http://parsleyjs.org/)
 - [DOMPurify](https://github.com/cure53/DOMPurify)
+- [MDN web security](https://developer.mozilla.org/en-US/docs/Learn/Server-side/First_steps/Website_security)
 
 ### Supplemental Resources - Web Security Libraries and Practices
 - [OWASP Secure Coding Practices Quick Reference Guide](https://www.owasp.org/images/0/08/OWASP_SCP_Quick_Reference_Guide_v2.pdf) 
 - [Passport](http://passportjs.org/)
 - [OpenID client connect](https://github.com/IdentityModel/oidc-client-js)
 
+### Lesson 
+- First thing that you must remember, never trust your user. There will always be someone with malicious intent out there. To safeguard you need security. What do we mean by security on web?
+	- [Read through this Security Lesson Slideshow.](https://docs.google.com/presentation/d/1mNyxzYGW-6M5yxBfJsxiwb9lcko5sa91thlBPKTodMg/edit?usp=sharing) (15 min)
+	- [MDN web security](https://developer.mozilla.org/en-US/docs/Learn/Server-side/First_steps/Website_security)
+
+- Authentication vs Authorization vs Access Control:<br>
+[Read through this slideshow explaining the difference between Authentication and Authorization](https://docs.google.com/presentation/d/1iHXimPdzKOYpjhXC2Vh-8QmpG90PxHWBQ-gRb6k32zg/edit?usp=sharing).  It has a few slides with examples written in Ruby, but you'll get the gist of it.<br>
+In simple words:
+	- Authentication is about who somebody is.
+	- Authorisation is about what they're allowed to do.
+	- Once we know who a user is, and we know what authorisation level they have and what we should and should not give them access to, we need to physically prevent that user from accessing anything that they should not by Access Control.
+
+- Read this 7-min article: [A quick introduction to web security
+](https://medium.freecodecamp.org/a-quick-introduction-to-web-security-f90beaf4dd41).
+
 ### Common Mistakes / Misconceptions
 
-OWASP releases a regular list of the [top 10 most critical web application security risks](https://www.owasp.org/index.php/Category:OWASP_Top_Ten_Project). Here are the 2017 highlights:
+- Changing all GET requests to POST requests does not protect you from CSRF attacks.
 - Injection: validate everything before you give it to an interpreter! Here we focus on Javascript sanitization.
 - Broken Authentication and Session Management: practice this by using the above libraries. Better than passwords, try SAML.
 - XSS and CSRF: XSS is basically injection in the DOM, and you can't prevent CSRF until you've tackled all the XSS bugs. Try it at home with [Google](https://xss-game.appspot.com/) and [Excess XSS](http://excess-xss.com/).
```

Comment:
"basically and injection in the DOM" ... can you give an example?

Also, can you please define CSRF too and give an example?